pom.xml

```diff
@@ -118,8 +118,9 @@
         <!-- When upgrading ZK, edit docs and integration tests as well (integration-tests/docker-base/setup.sh) -->
         <zookeeper.version>3.4.14</zookeeper.version>
         <checkerframework.version>2.5.7</checkerframework.version>
-        <com.google.apis.client.version>1.25.0</com.google.apis.client.version>
-        <com.google.apis.compute.version>v1-rev214-1.25.0</com.google.apis.compute.version>
+        <com.google.apis.client.version>1.26.0</com.google.apis.client.version>
+        <com.google.apis.compute.version>v1-rev20190607-1.26.0</com.google.apis.compute.version>
+        <com.google.apis.storage.version>v1-rev20190523-1.26.0</com.google.apis.storage.version>
```

Comment:
Similar to what was previously done in `extensions-core/google-extensions/pom.xml`, does it make sense to have the `1.26.0` suffix value provided by the `com.google.apis.client.version` property?